miniapp/build.gradle

```diff
@@ -71,6 +71,9 @@ dependencies {
     kapt "io.github.rakutentech.manifestconfig:manifest-config-processor:$manifest_config"
     implementation "io.github.rakutentech.manifestconfig:manifest-config-annotations:$manifest_config"
 
+    implementation("androidx.sqlite:sqlite-ktx:$sqlite_db")
+    implementation "net.zetetic:android-database-sqlcipher:4.5.0"
```

Comment:
probably, we can remove those round brackets `(...)` from L74, and we can put `4.5.0` value in `build.gradle` in the root.

Reply:
Addressed